<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRoomWindow</class>
 <widget class="QDialog" name="BookRoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="doctorsComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDepartmentButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>156</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QComboBox" name="departmentsComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>156</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectDoctorTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>211</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Doctor:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmRoomButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Room</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectDepartmentTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>157</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Department:</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectRoomTypeTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>311</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Room Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectTimeTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>261</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Time:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timesComboBox">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>260</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>09:00 AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10:00 AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11:00 AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12:00 PM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>01:00 PM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02:00 PM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03:00 PM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="roomTypesComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Standard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Suit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VIP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="statusTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectRoomTypeButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="feesTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>351</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fees:</string>
   </property>
  </widget>
  <widget class="QLabel" name="feesDisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>351</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="discountTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>381</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount by Points:</string>
   </property>
  </widget>
  <widget class="QLabel" name="discountDisplay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="roomTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Room</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
